README.md

```diff
@@ -12,6 +12,9 @@ Topics include how to download and install prebuilt CheriBSD images, how to
 build your own images, how third-party packages work, and where to find
 further information and support.
 
+**The document describes CheriBSD as of the 22.05 release, unless explicitly
```

Comment:
I'd prefer if `22.05` was a variable from `book.toml` but I didn't find a way to include a variable here.

Reply:
The solution with `book.toml` and a preprocessor would result in README.md displaying `{{ version }}` when rendered on GitHub (not GitHub Pages). I decided to add comments reminding where a version is mentioned.